src/lib/GovernorTimelockCompound.sol

```diff
@@ -0,0 +1,257 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.6.0) (governance/extensions/GovernorTimelockCompound.sol)
+//
+// The PoolTogether Timelock is not compatible with the Governor Alpha Timelock interface. This
+// meant we had to fork the contracts that interact with the timelock and replace the Timelock
+// interface with the PoolTogether Timelock interface. Also, if PoolTogether decides to change the
+// Timelock in the future then it must conform to the PoolTogether Timelock interface.
+//
+// PoolTogether Timelock interface:
+// https://etherscan.io/address/0xB3a87172F555ae2a2AB79Be60B336D2F7D0187f0#code#F1#L306
+// Governor Alpha Timelock interface:
+// https://github.com/compound-finance/compound-protocol/blob/a3214f67b73310d547e00fc578e8355911c9d376/contracts/Governance/GovernorAlpha.sol#L320
+
+pragma solidity ^0.8.0;
+
+import {Governor} from "flexible-voting/src/GovernorCountingFractional.sol";
+import {SafeCast} from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import {IGovernorTimelock} from
+  "@openzeppelin/contracts/governance/extensions/IGovernorTimelock.sol";
+import {IPoolTogetherTimelock} from "src/interfaces/IPoolTogetherTimelock.sol";
+import {Timers} from "@openzeppelin/contracts/utils/Timers.sol";
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+import {IERC165} from "@openzeppelin/contracts/interfaces/IERC165.sol";
+import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
+
+/**
+ * @dev Extension of {Governor} that binds the execution process to a Compound Timelock. This adds a
+ * delay, enforced by
+ * the external timelock to all successful proposal (in addition to the voting duration). The
+ * {Governor} needs to be
+ * the admin of the timelock for any operation to be performed. A public, unrestricted,
+ * {GovernorTimelockCompound-__acceptAdmin} is available to accept ownership of the timelock.
+ *
+ * Using this model means the proposal will be operated by the {TimelockController} and not by the
+ * {Governor}. Thus,
+ * the assets and permissions must be attached to the {TimelockController}. Any asset sent to the
+ * {Governor} will be
+ * inaccessible.
```

Comment:
More arbitrary line breaks here, recommend taking a pass through both contracts for other instances

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/20/commits/7445ab9b7d3b04e10af14887db666cd78b0852f5